pkg/whitesource/configHelper.go

```diff
@@ -165,10 +165,10 @@ func (c *ConfigOptions) addBuildToolDefaults(config *ScanOptions, utils Utils) e
 		//ToDo: switch to gomod as dependency manager
 		"golang": {
 			{Name: "ignoreSourceFiles", Value: true, Force: true},
-			{Name: "go.resolveDependencies", Value: true, Force: true},
 			{Name: "go.ignoreSourceFiles", Value: true, Force: true},
 			{Name: "go.collectDependenciesAtRuntime", Value: false},
-			{Name: "go.dependencyManager", Value: "modules"},
+			{Name: "go.modules.resolveDependencies", Value: true, Force: true},
+			{Name: "go.modules.ignoreSourceFiles", Value: true, Force: true},
```

Comment:
```suggestion
```
I suggest to remove two lines since we no longer assume the dependencyManager to be 'modules' and the values are by default 'true' for these parameters

Reply:
With these changes, and because `resolveAllDependencies` is set to `false` by default, nothing will be checked.

Using `go.modules.*` appears to be a drop-in replacement for the former `go.dependencyManager=modules` setting.

Reply:
In this case go.modules.* can stay. But need to add includes parameter. 
{Name: "includes", Value: "**/*.lock **/*.yml **/*.json **/*.tsv"}

Reply:
```suggestion
			{Name: "go.modules.resolveDependencies", Value: true, Force: true},
			{Name: "go.modules.ignoreSourceFiles", Value: true, Force: true},
			{Name: "includes", Value: "**/*.lock **/*.yml **/*.json **/*.tsv"},
```
I agree with @raghunathd8. The change should include the `includes` parameter to make it complete.